### PR TITLE
Correct typo on reduce memory tutorial

### PR DESCRIPTION
**Pull request by EduPonz**
Signed-off-by: EduPonz <eduardoponz@eprosima.com>

### CODE REVIEW

File: docs/fastdds/use_cases/reduce_memory/reduce_memory.rst [hunk 1]
```diff
@@ -21,12 +21,12 @@ Limiting Resources
 ^^^^^^^^^^^^^^^^^^^^^^^
 The :ref:`resourcelimitsqospolicy` controls the resources that the service can use in order to meet the requirements
 imposed. It limits the amount of allocated memory per :ref:`dds_layer_publisher_dataWriter` or
-:ref:`dds_layer_publisher_dataWriter`, as per the following parameters:
+:ref:`_dds_layer_subscriber_dataReader`, as per the following parameters:
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/use_cases/reduce_memory/reduce_memory.rst:24]:
```suggestion
:ref:`dds_layer_subscriber_dataReader`, as per the following parameters:
```

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]

File: docs/fastdds/use_cases/reduce_memory/reduce_memory.rst (hunk 1 above)
Comment (EduPonz) [reply docs/fastdds/use_cases/reduce_memory/reduce_memory.rst:24]: 🤦🏼‍♂️

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/801/)

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM!

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/802/)

**Pull request merged**